Heffner_Daniel/Rails Section 3/Ajax_Notes/app/views/notes/create.js.erb

```diff
@@ -0,0 +1 @@
+$("<%= escape_javascript(render @note) %>").appendTo("#notes");
```

Comment:
What does rendering the note itself do? Usually you render view files not objects.